test/integration/smoke/test_primary_storage.py

```diff
@@ -711,4 +854,4 @@ def test_03_migration_options_storage_tags(self):
         self.debug("Suitable storage pools found: %s" % len(pools_suitable))
         self.assertEqual(0, len(pools_suitable), "Check that there is no migration option for volume")
 
-        return
+        return
```

Comment:
The pre-commit lint is failing because of the removed line at the EOF.

Reply:
Thanks for pointing it. Fixed